### DISCUSSION TITLE
这周你做了什么？2022-05-06

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/emacstalk.github.io#17] [Show and tell]
- 可以随便讨论本周你打算做什么与 Emacs 相关的事情，甚至可以寻求帮助或反馈。
- 请记住，什么都不做也没关系！
- Happy Hacking Emacs！

**Comment by jiacai2050** [comment]
- 五一假期没怎么写代码，主要是把之前 C 的经验总结了下：https://liujiacai.net/blog/2022/04/30/modern-c/
- 发到 [v2ex](https://www.v2ex.com/t/850384#reply48) 上以为没人会看 C 的东西，没想到有这么多，有点意外：
- 
<img width="793" alt="image" src="https://user-images.githubusercontent.com/3848910/167238133-4fcfb4d4-c66d-4aa7-8314-3d299cd6886f.png">

**Reply by rever1and** [reply]
这周看了看21 centuries c然后看到了这个。。

**Comment by qiandu2006** [comment]
公司这周开始996了。。。

**Reply by jiacai2050** [reply]
貌似你这个和 Emacs 无关。。。是想让道友们帮你 run 嘛，懒猫的公司在招人

**Reply by calsys456** [reply]
摸摸～

**Reply by zwpaper** [reply]
用 Emacs 996 （Doge

**Comment by calsys456** [comment]
想去重写cnhl，可是我好懒啊，哪怕网课也压根不想动，写完作业就想上床大睡特睡😂

**Comment by calsys456** [comment]
刚写完cndict，帖子底下有同学说sdcv也可以做汉语词典，但我找了一圈愣是没找到汉语的sdcv词典，就，忽然又有点想玩玩，但没找到，憋得难受，有点想自己上手搞一个，flag立这了😂

**Comment by calsys456** [comment]
学校研学课要写论文，我选了个历史的课，人巨多，老师也摆烂让我们野蛮生长，然后给了我们一个学期时间交一篇历史论文，然后我就把这事忘了🤣 现在还有一星期我们组就要答辩了，然后我就想有没有一个领域对我来说既好写、能很快水出几千字，还研究人数少、竞争少。最开始我以为我在想桃子，直到我发现——诶，我可以写emacs发展史啊！

诸君请看我下周能不能活下来🤣

**Reply by jiacai2050** [reply]
哈哈，期待

**Reply by qiandu2006** [reply]
期待观摩一下

**Reply by rever1and** [reply]
自由软件运动史？

**Reply by calsys456** [reply]
> 自由软件运动史？

唔，其实并没有主要写自由软件运动，因为学术界对自由软件运动不感兴趣，反而一直在提开源，不仅在国家2035远景战略里是这样的，国家级的研究甚至是把GPL放在“开源”的话题下研究的。明显国家对自由软件运动不太感冒😂

主要还是emacs本身的发展，从TECO到GNU，以及什么Gosling emacs啊XEmacs啊这些的。其实不太敢发出来，因为班门弄斧😂 坛子里有位大佬，我不说是谁;)

老师都是外行，搞得我整篇文章写的其实很没意思，净让写些开源软件应该怎样发展什么的假大空的东西，真正有意义的东西跟我说听不懂，你赶紧略过去😂 嘛，写的开心就好，我管他呢，反正高中生写论文，写成啥样他都给过关～那还不如自由发挥一波，写点有用的东西

**Comment by flygerzy** [comment]
作为一个断断续续的vimer，最近开始折腾emacs，由于工作关系，大部分都是win下，发现win下emacs性能跟linux下差太多，慢慢折腾，当个文本编辑器用吧~毕竟生命在于折腾

**Reply by calsys456** [reply]
诶，win党+1（其实是win mac linux三教并行😂 ），确实巨慢，加油加油！
